core/broadleaf-framework-web/src/main/java/org/broadleafcommerce/core/web/controller/cart/BroadleafCartController.java

```diff
@@ -350,7 +350,7 @@ public String remove(HttpServletRequest request, HttpServletResponse response, M
 
     @Deprecated
     public String remove(HttpServletRequest request, HttpServletResponse response, Model model,
-                         AddToCartItem itemRequest) throws IOException, PricingException, RemoveFromCartException {
+                         ConfigurableOrderItemRequest itemRequest) throws IOException, PricingException, RemoveFromCartException {
```

Comment:
This needs to be OrderItemRequestDTO instead. First of all for compatibility reasons (i.e. clients may be using this method and sending it AddToCartItem which subclasses OrderItemRequestDTO however it doesn't subclass ConfiguratbleOrderItemRequest) and second of all if we only need it to be an OrderItemRequestDTO then there's not reason to require a more specific implementation